### PR TITLE
[ios] Fix the bottom 'download all' view layout

### PR DESCRIPTION
**Pull request by kirylkaveryn**
PR https://github.com/organicmaps/organicmaps/pull/7236 partly fixes the DownloadMapsVC layout but there are still some problems.

This PR fixes the Download/Update view on the DownloadMapsVC.

Before PR7236:
IMG_4237
IMG_4236

After PR7236(for screenshots I forced visibility of the view to show the layout and there is no content because the maps were up-to-date):
image
image

After this PR:
image

image

### CODE REVIEW

File: iphone/Maps/Categories/UIView+AddSeparator.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
+extension UIView {
```
Comment (biodranik) [inline iphone/Maps/Categories/UIView+AddSeparator.swift:1]: What are the benefits of introducing an extension that is used only in one place? Wouldn't it be better to localize this code only in the class that uses it?

Comment (kirylkaveryn) [reply iphone/Maps/Categories/UIView+AddSeparator.swift:1]: This extension was used 2nd time here (previous one in PP screen), so i made it non-private.

**Comment by biodranik** [comment]
There's too much empty space at the bottom now, it doesn't look good:

выява

**Comment by kirylkaveryn** [comment]
@biodranik are you talking about this space? 
image

**Comment by biodranik** [comment]
No, about the space between Update Maps and their size.

**Comment by kirylkaveryn** [comment]
> No, about the space between Update Maps and their size.
@biodranik fixed! 
Is the spacing above Download maps not too big? 

image
image
image

**Review by biodranik — APPROVED** [review]
Ok, let's see it in production.

**Pull request merged**